CMakeLists.txt

```diff
@@ -19,8 +19,8 @@ macro(build_yaml_cpp)
     list(APPEND extra_cmake_args -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE})
   endif()
   if(NOT WIN32)
-    list(APPEND YAML_C_FLAGS "-w")
-    list(APPEND YAML_CXX_FLAGS "-std=c++14 -w")
+    set(YAML_C_FLAGS "${YAML_C_FLAGS} -w")
+    set(YAML_CXX_FLAGS "${YAML_CXX_FLAGS} -std=c++14 -w")
```

Comment:
I'm fine with this in concept.  However, instead of using `YAML_C_FLAGS` and `YAML_CXX_FLAGS`, can we just remove lines 15 and 16 above, and instead just make this:

```
set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -w")
set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14 -w")
```

And use those below?

Reply:
sure, done in c5ffecf28c7bec82976b0be4369e73bbc028dda6